buildSrc/src/main/java/Dependencies.kt

```diff
@@ -77,4 +84,13 @@ object Deps {
     const val tools_androidgradle = "com.android.tools.build:gradle:${Versions.android_gradle_plugin}"
     const val tools_kotlingradle = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlin}"
     const val tools_appservicesgradle = "org.mozilla.appservices:gradle-plugin:${Versions.appservices_gradle_plugin}"
+
+    const val tools_espresso_contrib = "com.android.support.test.espresso:espresso-contrib:${Versions.tools_espresso_version}"
+    const val tools_espresso_core = "com.android.support.test.espresso:espresso-core:${Versions.tools_espresso_core}"
+    const val tools_espresso_idling_resources = "com.android.support.test.espresso:espresso-idling-resource:${Versions.tools_espresso_version}"
+    const val tools_espresso_web = "com.android.support.test.espresso:espresso-web:${Versions.tools_espresso_version}"
+    const val tools_test_orchestrator =  "com.android.support.test:orchestrator:${Versions.tools_test_orchestrator}"
+    const val tools_test_rules = "com.android.support.test:rules:${Versions.tools_test_rules}"
+    const val tools_test_runner = "com.android.support.test:runner:${Versions.tools_test_runner}"
+    const val tools_uiautomator = "com.android.support.test.uiautomator:uiautomator-v18:${Versions.tools_uiautomator}"
```

Comment:
Let's file a follow-up to migrate to the new `androidx.test` artifacts here (e.g. `androidx.test.espresso`)!

https://dl.google.com/dl/android/maven2/index.html